rmw_fastrtps_cpp/include/rmw_fastrtps_cpp/TypeSupport_impl.h

```diff
@@ -49,6 +49,56 @@ SPECIALIZE_GENERIC_C_ARRAY(uint32, uint32_t)
 SPECIALIZE_GENERIC_C_ARRAY(int64, int64_t)
 SPECIALIZE_GENERIC_C_ARRAY(uint64, uint64_t)
 
+// TODO(mikaelarguedas) find right spot to define these
```

Comment:
Done in https://github.com/eProsima/ROS-RMW-Fast-RTPS-cpp/pull/58/commits/07757844e03acd018a4abe54005b271240cd001c